### DISCUSSION TITLE
resize img

### DISCUSSION THREAD

**Discussion by flaggp** [libvips/libvips#2914] [Q&A]
can we convert a given png file to webp of size 1200 in one command?

@jcupitt we can do that first by using vips copy a.png b.webp then we can use vips resize command..but can we do that in a single command?

**Comment by jcupitt** [comment]
Hi @flaggp,

You can use any format as input or output, eg.:

```
vips resize x.png y.webp 0.2
```

You can also use `vipsthumbnail` to resize to a target size, eg.:

```
vipsthumbnail x.png -o y.webp -s 1200
```
